gammapy/irf/edisp/kernel.py

```diff
@@ -233,7 +233,10 @@ def from_hdulist(cls, hdulist, hdu1="MATRIX", hdu2="EBOUNDS"):
                     chan_max = l.field("F_CHAN")[k] + l.field("N_CHAN")[k]
 
                     pdf_matrix[i, chan_min:chan_max] = l.field("MATRIX")[
-                        m_start : m_start + l.field("N_CHAN")[k]  # noqa: E203
+                        m_start : m_start
+                        + l.field("N_CHAN")[
+                            k
+                        ]  # noqa: E203
```

Comment:
Was this changed by the precommit hook despite the noqa annotation? I think it is better as it was.

Reply:
Yes it must have been, I did not adjust that